custom_otello_linter/errors/errors.py

```diff
@@ -3,7 +3,7 @@
 
 class DecoratorVedroParams(Error):
     code = 'OCS101'
-    message = 'decorator @vedro.params should not be presented'
+    message = 'decorator @vedro.params or @params should not be presented'
```

Comment:
This part is misleading. `@params` decorator is in fact allowed to be used (a custom one, for instance), just not the one from the `vedro` package.
> or @params should not be presented

I suggest:
"params decorator from vedro package should not be present"